### PR TITLE
[1764] Accessibility testing and semantic updates

### PR DESCRIPTION
**Pull request by markmead**
### Context

Fixed accessibility issues and tidy up semantic HTML

### CODE REVIEW

File: app/views/courses/show.html.erb [hunk 1]
```diff
@@ -73,18 +73,18 @@
...
-        class: "govuk-tabs__tab govuk-tabs__tab--selected", id: "description", role: 'tab', tabindex: '-1', aria: { selected: true } do
+        class: "govuk-tabs__tab govuk-tabs__tab--selected", role: 'tab', tabindex: '-1', aria: { selected: true } do
       %>
         Description<br>
         <span class="govuk-body-s govuk-!-font-weight-regular pointer-events-none">Content, fees and eligibility</span>
       <% end %>
     </li>
 
-    <li class="govuk-tabs__list-item">
+    <li class="govuk-tabs__list-item" role="presentation">
```
Comment (fofr) [inline app/views/courses/show.html.erb]: Why is this role needed?

It's not in the design system component, which will have been rigorously tested for accessibility: https://design-system.service.gov.uk/components/tabs/

File: app/views/courses/show.html.erb [hunk 2]
```diff
@@ -101,9 +101,9 @@
         <%= render partial: 'courses/description_content' %>
       </div>
       <% unless course.not_running? %>
-        <div class="govuk-grid-column-one-third" data-qa="course__status_panel">
+        <aside class="govuk-grid-column-one-third" data-qa="course__status_panel">
```
Comment (fofr) [inline app/views/courses/show.html.erb]: I don't think we should be switching from `div` to `aside`.

Aside:
> a portion of a document whose content is only indirectly related to the document's main content

I would not say that the content of these side columns is indirectly related to the document. In the case of the status panel it's crucial.

File: app/views/courses/show.html.erb [hunk 3]
```diff
@@ -73,18 +73,18 @@
 <div class="course-tabs govuk-tabs">
   <h2 class="govuk-tabs__title">Contents</h2>
 
-  <ul class="govuk-tabs__list">
+  <ul class="govuk-tabs__list" role="tablist">
```
Comment (fofr) [inline app/views/courses/show.html.erb]: I don't think this role is needed, it's not included in the design system, see also: https://simplyaccessible.com/article/danger-aria-tabs/

File: app/views/courses/show.html.erb (hunk 1 above)
Comment (markmead) [reply app/views/courses/show.html.erb]: [I wasn't quite sure if it was necessary but it existed on the sibling `li`](https://github.com/DFE-Digital/manage-courses-frontend/blob/b209a35930b2ea2f499d7ec5db9c99ef70ed948d/app/views/courses/show.html.erb#L77)

Comment (fofr) [reply app/views/courses/show.html.erb]: Probably best to remove them all

Comment (fofr) [reply app/views/courses/show.html.erb]: Ah, that’s probably because it’s on the currently selected tab, where the link isn't particularly useful.

Comment (markmead) [reply app/views/courses/show.html.erb]: So leave there and remove on the other?

**Comment by fofr** [comment]
[Accessibility testing](https://trello.com/c/wSLsEX5E/1764-accessibility-testing)

**Pull request merged**